internal/pkg/manifest/validate.go

```diff
@@ -846,6 +856,28 @@ func (c NetworkLoadBalancerConfiguration) validate() error {
 	if c.IsEmpty() {
 		return nil
 	}
+	if err := c.Listener.validate(); err != nil {
+		return fmt.Errorf("validate nlb: %w", err)
```

Comment:
Should "validate nlb" be prefixed by functions that call `NetworkLoadBalancerConfiguration.validate()` 🤔

Reply:
yes actually I was thinking the same how we can have a different error message in here because else we will end up having `validate "nlb": validate nlb` for main listener.

Reply:
The `validate "nlb"` wrapping already happens at https://github.com/aws/copilot-cli/blob/7c00353d9c743d55b1490479e134e3c892083e5e/internal/pkg/manifest/validate.go#L234. I think we can remove all "validate nlb" wrapping in this function. E.g.

```go
if err := c.Aliases.validate(); err != nil {
		return fmt.Errorf(`validate "alias": %w`, err)
	}
```